Automattic/jetpack
# Contact Form: Fix empty `from` field in admin panel

Default 'from' column in the contact forms admin panel to IP address, if no name or email is provided on submission. 

<img width="876" alt="Screenshot 2023-03-15 at 3 46 30 PM" src="https://user-images.githubusercontent.com/18619437/225451372-57d7dcc6-ae70-48b2-9c81-cdcb4741b9b4.png">





Fixes #28276

## Proposed changes:

* Changed the if statements in the function `grunion_manage_post_column_from` to use !empty() instead of isset() to account for blank ('') database columns.
* Fixed $content_fields feedback_ip field. Missing the leading underscore. 

### Other information:


## Does this pull request change what data or activity we track or use?


No

## Testing instructions:






1. Create a contact form
2. Make 'from' name and 'email' not required
3. Submit a blank form
4. Check the Admin Contact Form panel
